pinkie-card/GenericTileIcon.qml

```diff
@@ -0,0 +1,135 @@
+import QtQuick 2.7
```

Comment:
Yup, I don't know how to make proper inheritance without having to write a lot more code (with C++ bindings and weird things) :/